aws_lambda_builders/binary_path.py

```diff
@@ -0,0 +1,22 @@
+"""
+Class containing resolved path of binary given a validator and a resolver and the name of the binary.
+"""
+
+
+class BinaryPath(object):
+
+    def __init__(self, resolver, validator, binary, binary_path=None):
+        self.resolver = resolver
+        self.validator = validator
+        self.binary = binary
+        setattr(self, binary, binary)
```

Comment:
This is equivalent to the line above right?

Reply:
No.

```
In [1]: class A(object):
   ...:     def __init__(self, a):
   ...:         self.a = a
   ...:         setattr(self, a , a)
   ...:

In [2]: A(a="python")
Out[2]: <__main__.A at 0x109ebcd68>

In [3]: obj = A(a="python")

In [4]: obj.a
Out[4]: 'python'

In [5]: obj.python
Out[5]: 'python'
```